docs/docs/guides/remove-offline-files.md

```diff
@@ -15,13 +146,13 @@ This way works by downloading a JSON file that contains a list of all the files
 
 ## Script for Linux based systems:
 
-```bash
+```bash title='Bash'
 awk -F\" '/entityId/ {print $4}' orphans.json | while read line; do curl --location --request DELETE 'http://YOUR_IP_HERE:2283/api/asset' --header 'Content- Type: application/json' --header 'x-api-key: YOUR_API_KEY_HERE' --data '{ "force": true, "ids": ["'"$line"'"]}';done
 ```
 
 ## Script for the Windows system (run through PowerShell):
 
-```powershell
+```bash title='PowerShell'
```

Comment:
I believe we want the language to be `powershell`, not `bash`. Did you change it on purpose?

Reply:
Oh I see, I got it wrong.
fixed